pages/[slug].page.tsx

```diff
@@ -30,7 +30,7 @@ export default function StaticMarkdownPage({
       </Head>
       <Headline1>{frontmatter.title}</Headline1>
       <StyledMarkdown markdown={content} />
-      <DocsHelp />
+      <DocsHelp githubLink='https://github.com/json-schema-org/website/blob/main/pages/%5Bslug%5D.page.tsx' />
```

Comment:
We should try to make this dynamic and avoid this kind of static arguments. I think it should be possible without this type argument.